src/Core/Form/Form.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+
+import "./_form.scss";
```

Comment:
Let's be consistent about placing style imports before or after component imports. In other places, you're putting style import after component imports. Let's do the same in here to or vice versa.